css/styles.css

```diff
@@ -1,52 +1,57 @@
-* {
+*, *::before, *::after {
 	box-sizing: border-box;
 }
 
+:root {
+	font-size: 15px;
+}
+
 body {
-	background-color: #fff;
+	background: #fff;
+	color: #000;
 }
 
 a {
 	text-decoration: none;
 	color: rgba(71, 71, 107, .7);
 }
 
+a:hover, a:focus {
+	color: #000;
+}
+
 .container {
 	max-width: 1200px;
+	padding: 0 20px;
 	margin: 0 auto;
 	text-align: center;
 }
 
 /* header and navigation menu */
 
-.minimo {
+.logo {
 	padding-top: 30px;
 	padding-bottom: 20px;
 }
 
-.main-nav ul {
-	margin: 0 auto;
-}
 
 .main-nav li {
-	font-family: "Playfair Display", serif;
+	padding: 5px 0;
 	color: rgba(71, 71, 107, .7);
-	font-size: 0.8rem;
-	font-weight: 500;
+	font: 0.8rem "Playfair Display", serif;
 	text-transform: uppercase;
-	padding: 5px 0;
+}
+
+.main-nav li:last-child {
+	padding-right: 0;
 }
 
 .header-photo {
-	background-image: url("../images/Header_photo.jpg");
-	background-repeat: no-repeat;
-	background-position: center;
-	height: 600px;
+	margin-bottom: 60px;
 }
 
 .header-photo img {
-	margin: 0 auto;
-	width: 90%;
+	max-width: 100%;

```

Comment:
Zweryfikuj szerokość kontenera jaką masz i tą z PSD-ka. Bo u Ciebie to zdjęcie się nie rozciąga na całą szerokość kontenera. Możesz też dać `width: 100%;` zamiast maksymalnej szerokości. Lepiej IMO to wygląda.